tasks/run_agent.rb

```diff
@@ -0,0 +1,160 @@
+#!/opt/puppetlabs/puppet/bin/ruby
+
+require 'open3'
+require 'etc'
+require 'json'
+require 'yaml'
+require_relative '../../ruby_task_helper/files/task_helper.rb'
+
+class RunAgentTask < TaskHelper
```

Comment:
No idea what to use for the class name.  The [docs](https://forge.puppet.com/puppetlabs/ruby_task_helper#usage) aren't especially helpful.